### ISSUE TITLE
Memory leak in PyTorch DataLoader with custom collate function

### ISSUE THREAD

**Issue by DimitriFrischmann** [DimitriFrischmann/Spelter#69]
The API reference for the protein-folding utility does not specify the units for the atomic-distance constraints. It's unclear if we are using Angstroms or nanometers.

**Comment by DimitriFrischmann** [comment]
Closes #70 Fix subtle race condition in async scheduler. Improved thread affinity binding. Documented workaround for flaky CI job
